index.html

```diff
@@ -115,6 +115,46 @@ <h2>A project for first-time contributions</h2>
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
 
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
+        <div class="card">
+          <p class="name">Rodolpho Napoleon</p>
+          <p class="contact">
+            <i class="fab fa-linkedin"></i>
+            <a href="https://www.linkedin.com/in/rodolpho-richard-napoleon-30337113/" target="_blank">LinkedIn</a>
+          </p>
+          <p class="about">
+            I love learning new skills and programming languages. I already have under my belt JavaScript, React,
+            ExpressJS, NodeJS, MongoDB, Graphql, HTML, CSS, Bootstrap, Git/GitHub. I’m learning actually the MERN stack
+            at MITxPro. I’m enthusiastically grabbing onto any other programming languages, frameworks, or principles I
+            can integrate into the coding web in my head.
+          </p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="https://developer.mozilla.org/en-US/" target="_blank" title="A link to learn web development">
+                  MDN
+                </a>
+              </li>
+              <li>
+                <a
+                  href="https://stackoverflow.com/"
+                  target="_blank"
+                  title="A Platform to aske questions and help others"
+                >
+                  Stack Overflow
+                </a>
+              </li>
+              <li>
+                <a href="https://www.w3schools.com/" target="_blank" title="A platform to learn">
+                  W3Schools
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*END TEMPLATE* Contributor card END ________  -->
```

Comment:
Please change this comment to 
`<!-- ________ Your Name Contributor card END________ -->`